discovery_engine_core/web-api/src/migrations/20220908090717_coi.sql

```diff
@@ -12,7 +12,12 @@
 --  You should have received a copy of the GNU Affero General Public License
 --  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-CREATE TABLE IF NOT EXISTS user_state (
-    id TEXT NOT NULL PRIMARY KEY,
-    state BYTEA NOT NULL
+CREATE TABLE IF NOT EXISTS center_of_interest (
+    coi_id UUID NOT NULL PRIMARY KEY,
+    user_id TEXT NOT NULL,
```

Comment:
:wrench:  Shouldn't this have  be a foreign key constraint? 

Preferable with `on delete cascade` so that when we delete the user it always automatically deletes all cois of the user.

Reply:
it's not a foreign key, because we don't store users as separate entities, the `user_id` here is just a key to filter the CoIs. Do we need to store users in a separate table? @acrrd

Reply:
from daily: no user table.

:wrench: We need  an index on `user_id` to make sure filtering by `user_id` is efficient.

Reply:
added, please check :)